### ISSUE TITLE
Torch protection applied to wrong save

### ISSUE THREAD

**Issue by KikiCur** [Xalcon/TorchMaster#53]
Hi, Xalcon,

Been playing Forever Stranded Lost Souls and hopping from my test world to my play world. Just now I placed 9 torches in a fresh test world and then went back to the play world. Over the next half hour or so, I noticed mobs just stopped spawning. I did a quick fly around and they're no longer spawning over the same area that was protected on my test world.
I logged into my test world to double check where the torches were and back in play world to make sure there are none placed in that one and that's all in order.
So basically it seems the torches' protection is being applied to the wrong save somehow O_o?
Thanks tons for your time.

Versions
TorchMaster: 1.6.1.53
Forge: 14.23.2.2618
MC: 1.12.2

**Comment by Spazley** [comment]
I had the same issue.  Confirmed it was related to megatorches by changing the megatorch radius to 0, killing all mobs, and letting mobs respawn.  The new mobs were within the area previously without any spawns.  After changing the value back, killing all mobs, and letting mobs respawn, the new mobs were limited to spawning outside the "ghost" megatorch's range.

Placing a megatorch in the same location as the megatorch in the other save then breaking it appears to have corrected the issue.

**Comment by Xalcon** [comment]
Just to clarify, you didnt restart the game between those 2 worlds? I was able to track this bug down to the registry not being cleared between world loads. This will be fixed in an upcoming release.
